openhub/oh_token.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+from gci.config import get_api_key
+
+try:
+    OH_TOKEN = get_api_key('OH')
+except Exception as ex:
+    logging.debug('OH_TOKEN can not be obtained: \n{}'.format(str(ex)))
```

Comment:
also set `OH_TOKEN = None` , otherwise this module has an inconsistent set of members.